press/article.md

```diff
@@ -0,0 +1,85 @@
+# Introduction
+
+In the recent react project we needed an environment to create, style and test our components without having a server-side backend code. We decided to use storybook that allows us to develop UI components outside your app and allow other people in your team to work on them. 
+
+The most important benefits of using storybook are :
+1. Completely isolate the environment for your components
+2. HMR — hot module replacement
+3. Support for CSS
+4. Multiple add-ons like accessibility, readme etc
```

Comment:
enumarate accesibility, readme, snapshot testing of React components, CSS regression and make the texts clickable(links). For CSS regression use the following https://github.com/tsuyoshiwada/storybook-chrome-screenshot /